sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -74,9 +84,12 @@ first_session_ping AS (
     ] AS adjust_creative
   FROM
     `moz-fx-data-shared-prod.fenix.first_session` AS fenix_first_session
+  INNER JOIN
```

Comment:
The goal is to get all first_session pings and we use the seq to identify when the ping was generated on the client side rather than when it is received by the server. The INNER JOIN would exclude clients if we ever receive pings with seq = NULL and it would be a silent error if it happens. Likely a left join would solve it? Else  we may need another approach to retrieve the ping with the minimum or NULL sequence.